.travis.yml

```diff
@@ -11,11 +11,11 @@ env:
     - TVOS_FRAMEWORK_SCHEME=Hyperspace_tvOS
     - WATCHOS_FRAMEWORK_SCHEME=Hyperspace_watchOS
   matrix:
-    - DESTINATION="platform=iOS Simulator,OS=11.2,name=iPhone X"                        SCHEME="$IOS_FRAMEWORK_SCHEME"      RUN_TESTS="YES" POD_LINT="NO"
+    - DESTINATION="platform=iOS Simulator,OS=11.2,name=iPhone X"                        SCHEME="$IOS_FRAMEWORK_SCHEME"      RUN_TESTS="YES" POD_LINT="YES"
 
-    - DESTINATION="platform=tvOS Simulator,OS=11.2,name=Apple TV 4K"                    SCHEME="$TVOS_FRAMEWORK_SCHEME"     RUN_TESTS="YES" POD_LINT="NO"
+    - DESTINATION="platform=tvOS Simulator,OS=11.2,name=Apple TV 4K"                    SCHEME="$TVOS_FRAMEWORK_SCHEME"     RUN_TESTS="YES" POD_LINT="YES"
```

Comment:
`pod spec lint` actually only needs to be run once per build so let's remove it from the tvOS and watchOS targets so there's no redundancy. Proactively approving changes...

Reply:
Fixed